### DISCUSSION TITLE
support deploy production when release a version

### DISCUSSION THREAD

**Discussion by himself65** [vercel/vercel#8625] [General] [answered]
Our team is using GitHub releases to publish a new version automatically. I want to trigger a production deployment using the GitHub release instead of the GitHub branch

![Screenshot from 2022-09-23 17-10-41](https://user-images.githubusercontent.com/14026360/192063328-0eb4a566-5b58-45b8-961d-532de2799480.png)

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @Himself65! I think that customizing GitHub Actions for your workflow would give you the best control over this. We have a guide on [using GitHub Actions with Vercel](https://vercel.com/guides/how-can-i-use-github-actions-with-vercel) that can walk you through basic setup. Then, you can customize it to use the [release trigger](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release) so a deployment happens automatically with each new GitHub release.

**Reply by himself65** [reply]
Thanks for the reply. I do this way to release production now. It's more flexible.
